packages/vue3/package.json

```diff
@@ -22,6 +22,7 @@
   "files": [
     "dist"
   ],
+  "types": "types/index.d.ts",
```

Comment:
I don't see it exists

Reply:
The pipeline will copy the types from `packages/vue2` before publishing.